docs/oauth.md

```diff
@@ -0,0 +1,31 @@
+# OAuth2
+
+## Login with Discord using OAuth2 protocol
+
+### Instructions
+
+1. Initiate the login process by accessing the following URL from the frontend application: 
```

Comment:
What do you mean by access? Is this a redirect?

Reply:
yes. The login button would be actually a link

```html
<a href="<DISCORD_OAUTH_URL>">Login</button>
```

Reply:
Makes sense!